vpc/service/reconcile_branch_enis.go

```diff
@@ -351,43 +351,6 @@ retry:
 
 	span.AddAttributes(trace.StringAttribute("networkInterfaceSecurityGroups", fmt.Sprintf("%+v", networkInterfaceSecurityGroups)))
 
-	_, err = fastTx.ExecContext(ctx, "UPDATE branch_enis SET mac = $1 WHERE branch_eni = $2 AND mac IS NULL",
```

Comment:
Why is it that we dont need `mac` anymore?

Reply:
Mac was a column that was added a while ago. The original table didn't have MAC in it. The reconciler was supposed to "fix" that -- it's been in prod long enough where that should be set now.

Now, ENIs should have MAC addresses set when inserted into the DB.